BlazarService/src/main/java/com/hubspot/blazar/guice/BlazarServiceModule.java

```diff
@@ -68,29 +68,29 @@
   private static final Logger LOG = LoggerFactory.getLogger(BlazarServiceModule.class);
 
   /**
-   *  Blazar has an option to enable "webhook only" mode. This is because you may wish to have
-   *  a public facing Blazar instance that does not have the ability to start builds etc. But can still
-   *  receive and process webhooks from external GitHub installations (like GitHub.Com).
+   * Blazar has an option to enable "webhook only" mode. This is because you may wish to have
+   * a public facing Blazar instance that does not have the ability to start builds etc. But can still
+   * receive and process webhooks from external GitHub installations (like GitHub.Com).
    *
    * In webhook only mode we configure only the bare minimum required resources for Blazar to be able
    * to accept web-hook events and send them into our SQL backed event bus. Other Blazar instances running
-   * against the same database are then able to process those events using the SQL backed event bus.
-   *
+   * against the same database are then able to process those events using the SQL backed event bus. These
+   * instances are also configured to accept webhooks, but also have the rest of Blazar's API enabled.
    */
   @Override
   public void configure(Binder binder) {
     BlazarConfiguration blazarConfiguration = getConfiguration().getBlazarConfiguration();
-    configureBase(binder, blazarConfiguration);
+    configureWebhooks(binder, blazarConfiguration);
```

Comment:
@gchomatas I re-worked this to work like we discussed offline. To bind all the webhook stuff and then bind extra stuff only if we're not in webhook mode.